lets-do-more-for-it/before-the-hard-part.md

```diff
@@ -1,21 +1,21 @@
 # Before the hard part
 
-Everyone reading this chapter of the guide book must be not satisfied by the functions a-Shell already has, and at this chapter we will talk about various possibilities of what this App can do.
+Everyone reading this chapter of the guide book must not be satisfied by the functions a-Shell already has, and at this chapter we will talk about various possibilities of what this App can do.
```

Comment:
is "must not" here the meaning as "should not" but stronger?

Reply:
Yes.

"...must be not satisfied..." is grammatically incorrect for the current state of English. If you wanted it in that order, the "un" suffix would be attached to "satisfied" and the "not" would be removed. That would make the result "...must be unsatisfied...".